runbotics/runbotics-orchestrator-ui/src/src-app/translations/en/process/actions/folder.ts

```diff
@@ -1,15 +1,20 @@
 const folderActionsTranslations = {
-    'Process.Details.Modeler.Actions.Folder.Delete.Label': 'Delete folder',
+    'Process.Details.Modeler.Actions.Folder.Delete.Label': 'Delete',
     'Process.Details.Modeler.Actions.Folder.Delete.Path': 'Path',
     'Process.Details.Modeler.Actions.Folder.Delete.Name': 'Name',
     'Process.Details.Modeler.Actions.Folder.Delete.Recursive': 'Recursive',
     'Process.Details.Modeler.Actions.Folder.DisplayFiles.Label': 'Display files',
     'Process.Details.Modeler.Actions.Folder.DisplayFiles.Path': 'Path',
     'Process.Details.Modeler.Actions.Folder.DisplayFiles.Name': 'Name',
-    'Process.Details.Modeler.Actions.Folder.Create.Label': 'Create folder',
+    'Process.Details.Modeler.Actions.Folder.Create.Label': 'Create',
     'Process.Details.Modeler.Actions.Folder.Create.Name': 'Name',
     'Process.Details.Modeler.Actions.Folder.Create.Name.Info': 'New folder name',
     'Process.Details.Modeler.Actions.Folder.Create.Output.Info': 'Save path of the newly created folder to a variable',
+    'Process.Details.Modeler.Actions.Folder.Rename.Label': 'Rename',
+    'Process.Details.Modeler.Actions.Folder.Rename.Path.Info': 'Path, e.g. C:\\Documents\\OldFolderName',
```

Comment:
imo add info what in case if path is not specified (is the current directory taken or what)

Reply:
Path is mandatory, so I don't think there needs to be any additional info